src/Entity/Comments.php

```diff
@@ -93,21 +98,11 @@ public function setPostId(int $postId): bool
     /**
      * @return string
      */
-    public function getCreatedBy(): string
+    public function getCreatedBy(): DateTime
     {
```

Comment:
To chyba ja Ci kazałem zmienić na DateTime, za co przepraszam, bo myślałęm, ze to created at. W każdym razie i tak, źle, bo powinno przechowywać relację do userów